pkg/safeguards/preprocessing/preprocessing.go

```diff
@@ -28,7 +30,7 @@ func GetManifestFiles(manifestsPath string) ([]sgTypes.ManifestFile, error) {
 	if isDir {
 		// check if Helm or Kustomize dir
 		if isHelm(true, manifestsPath) {
```

Comment:
I think the approach of trying to read from values.yaml if not specified is fine - if customers complain, then we can change it

Reply:
I think that makes sense, but I think people would compIain because the practice of specifying release name and namespace in the values.yaml file doesn't seem common. Currently, we would error if the user didn't provide flags and didn't think to include the release name and ns in values.yaml. In this case, wouldn't it be a better user experience to set it for them, as it better simulates how they would use helm otherwise?

Reply:
Included both approaches in the conditional logic.